redesign.user.js

```diff
@@ -797,14 +797,69 @@ const createSidebar = (id, position, icon, callback) => {
         document.getElementById('page')?.before(sidebar);
 
         // append the toggle button
-        document
-            .querySelector('#page .drawer-toggles')
-            ?.append(toggleBtnWrapper);
+        const togglesDiv =
+            document.querySelector(
+                `.${PREFIX(`drawer-toggles-${position}`)}`
+            ) ||
+            (() => {
+                const togglesDiv = document.createElement('div');
+                togglesDiv.classList.add(PREFIX(`drawer-toggles-${position}`));
```

Comment:
any particular reason why to use a class and not an ID?
As far as I understand, for each side, only a single toggle will exist, thus an ID would be more straightforward in my eyes.

Reply:
This is resolved with https://github.com/YorikHansen/better-moodle/commit/9729a5c8c323a96e582b65752262fab4875b9178

Reply:
resolved with https://github.com/YorikHansen/better-moodle/commit/28393e5ba4edcc2d727a8d2627367e3557483f4c